UnityProject/Assets/Scripts/Player/PlayerSync.cs

```diff
@@ -396,19 +396,36 @@ public void setLocalPlayer()
 		predictedSpeedClient = UIManager.WalkRun.running ? playerMove.RunSpeed : playerMove.WalkSpeed;
 	}
 
+	private bool didWiggle = false;
```

Comment:
this might need a summary or small comment

Reply:
yeah